hieradata/env/production.yaml

```diff
@@ -10,6 +10,7 @@ exim::relayhost: '*.mailgun.org'
 
 metacpan::carton_args: '--deployment'
 
+metacpan::elasticsearch::status: 'disable'
```

Comment:
Needs to be `disabled` according to https://github.com/metacpan/metacpan-puppet/blob/master/contrib-modules/elasticsearch/manifests/instance.pp#L109-L118